spec/Section 3 -- Type System.md

```diff
@@ -1554,8 +1554,10 @@ Input object type extensions have the potential to be invalid if incorrectly def
 A GraphQL list is a special collection type which declares the type of each
 item in the List (referred to as the *item type* of the list). List values are
 serialized as ordered lists, where each item in the list is serialized as per
-the item type. To denote that a field uses a List type the item type is wrapped
-in square brackets like this: `pets: [Pet]`.
+the item type.
+
+To denote that a field uses a List type the item type is wrapped in square brackets
+like this: `pets: [Pet]`. Nesting lists is allowed: `matrix: [[Int]]`.
```

Comment:
Maybe 

```suggestion
like this: `pets: [Pet]`. Nesting lists are allowed: `matrix: [[Int]]`.
```

Reply:
*Nested lists are allowed* would be correct, i used nesting as a verb though.

Reply:
ok